### PR TITLE
Use libpod GetDefaultStorage to report proper storage config

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@giuseppe PTAL

File: cmd/buildah/main.go [hunk 1]
```diff
@@ -19,6 +20,11 @@ func main() {
 	if buildah.InitReexec() {
 		return
 	}
+	storageOptions, err := util.GetDefaultStoreOptions()
```
Comment (giuseppe) [inline cmd/buildah/main.go]: I think this won't work in buildah, as it uses the `rootless` package from libpod while buildah uses its own unshare reexec module.

Comment (rhatdan) [reply cmd/buildah/main.go]: I'm at least seeing the proper content in the help.  And rootless seems to work.

```
$ ./buildah --help
...
   --root value                          storage root dir (default: "/home/dwalsh/.local/share/containers/storage")
   --runroot value                       storage state dir (default: "/run/user/3267/run")
```
Versus
```
$ buildah --help
...
   --root value                          storage root dir (default: "/var/lib/containers/storage")
   --runroot value                       storage state dir (default: "/var/run/containers/storage")
```

**Comment by giuseppe** [comment]
@rhatdan I've opened a PR that adds the missing pieces: https://github.com/containers/buildah/pull/1124

**Comment by TomSweeneyRedHat** [comment]
@giuseppe Do we need to merge #1123 before or after #1124 or does the merge order not matter?

**Comment by giuseppe** [comment]
this is a replacement for #1123, in facts it is based on top of it

File: docs/buildah.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ specify additional options via the `--storage-opt` flag.
 
 **--storage-opt** **value**
 
-Storage driver option, Default storage driver options are configured in /etc/containers/storage.conf. The `STORAGE_OPTS` environment variable overrides the default.  The --storage-opt specified options overrides all.
+Storage driver option, Default storage driver options are configured in /etc/containers/storage.conf (`$HOME/.config/containers/storage.conf` in rootless mode). The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options overrides all.
```
Comment (TomSweeneyRedHat) [inline docs/buildah.md]: nit of a nit, and not in your changes, but "Default storage" should be "default storage", or we should replace the preceding comma with a period and change to "The default" like in --storage-driver.  Ignoring or doing in a follow up is fine.

**Pull request closed without merging**